src/LibChorus/merge/xml/generic/MergeChildrenMethod.cs

```diff
@@ -57,6 +57,16 @@ public void Run()
 			var parentNode = _ours ?? _theirs ?? _ancestor;
 			var parentStrategy = _merger.MergeStrategies.GetElementStrategy(parentNode);
 			var parentOrder = parentStrategy == null ? ChildOrder.AskChildren : parentStrategy.ChildOrderPolicy.OrderSignificance(parentNode);
+			// Determine if child order is important, we shouldn't create reordering conflicts if the order of children is unimportant
+			var childOrderMatters = parentOrder == ChildOrder.Significant;
+			if(parentOrder == ChildOrder.AskChildren && parentNode.HasChildNodes)
+			{
+				var childStrategy = _merger.MergeStrategies.GetElementStrategy(parentNode.FirstChild);
+				if(childStrategy != null && childStrategy.OrderIsRelevant)
+				{
+					childOrderMatters = true;
```

Comment:
As I understand it, the generic xml does not support having children with different strategies. There are other places in this area that assume only the first child under a parent matters.